airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml

```diff
@@ -0,0 +1,64 @@
+connector_image: airbyte/source-stripe:dev
+tests:
+  spec:
+    - spec_path: "source_stripe/spec.json"
+  connection:
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "secrets/connected_account_config.json"
+      status: "succeed"
+    - config_path: "integration_tests/invalid_config.json"
+      status: "failed"
+  discovery:
+    - config_path: "secrets/config.json"
+    - config_path: "secrets/connected_account_config.json"
+  basic_read:
+    - config_path: "secrets/config.json"
+#     Reading a invoice_line_items stream takes too long and falls on timeout.
+      configured_catalog_path: "sample_files/non_invoice_line_items_catalog.json"
+      validate_output_from_all_streams: yes
+    - config_path: "secrets/connected_account_config.json"
+#     This account has no records for stream Disputes.
+      configured_catalog_path: "sample_files/non_disputes_catalog.json"
+      validate_output_from_all_streams: yes
+  incremental:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "sample_files/non_invoice_line_items_catalog.json"
+      state_path: "integration_tests/abnormal_state.json"
+      cursor_paths:
+        charges: [ "created" ]
+        coupons: [ "created" ]
+        customers: [ "created" ]
+        customer_balance_transactions: [ "created" ]
+        disputes: [ "created" ]
+        events: [ "created" ]
+        invoice_items: [ "date" ]
+        invoices: [ "created" ]
+        plans: [ "created" ]
+        payouts: [ "created" ]
+        products: [ "created" ]
+        subscriptions: [ "created" ]
+        refunds: [ "created" ]
+        transfers: [ "created" ]
+    - config_path: "secrets/connected_account_config.json"
+      configured_catalog_path: "sample_files/non_disputes_catalog.json"
+      state_path: "integration_tests/abnormal_state.json"
+      cursor_paths:
+        charges: [ "created" ]
```

Comment:
you don't need to specify a path if it is not different from cursor_field in schema